ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Individual/CheckingVideoHearing.feature

```diff
@@ -1,11 +1,11 @@
-﻿Feature: About Hearings
+﻿Feature: Checking Video Hearing
 	As a Participant
 	I want to be able to tell the court if there is any reason I don't think I am suitable
 	So that the court can make an informed decision based on my circumstances
 
 @Individual
-Scenario: About Hearings
-	Given the Individual has progressed to the About Hearings page
+Scenario: Checking video Hearing
+	Given the Individual has progressed to the Checking Video Hearing page
 	Then contact details are available
 	When the user clicks the Next button
 	Then the user is on the Check Your Computer page
```

Comment:
Can you put this into the ../Shared directory and add the Individual and Rep tests to the same file?

Reply:
I moved it to shared features